imagestreams/nginx-rhel7.json

```diff
@@ -11,47 +11,27 @@
     "tags": [
       {
         "annotations": {
-          "description": "Build and serve static content via Nginx HTTP server and a reverse proxy (nginx) on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/nginx-container/blob/master/1.10/README.md.",
+          "description": "Build and serve static content via Nginx HTTP server and a reverse proxy (nginx) on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/nginx-container/blob/master/1.14/README.md.",
           "iconClass": "icon-nginx",
-          "openshift.io/display-name": "Nginx HTTP server and a reverse proxy 1.10",
+          "openshift.io/display-name": "Nginx HTTP server and a reverse proxy 1.14",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
           "sampleRepo": "https://github.com/sclorg/nginx-ex.git",
           "supports": "nginx",
           "tags": "builder,nginx",
-          "version": "1.10"
+          "version": "1.14"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/nginx-110-rhel7:latest"
+          "name": "registry.redhat.io/rhscl/nginx-114-rhel7:latest"
         },
         "referencePolicy": {
           "type": "Local"
         },
-        "name": "1.10"
+        "name": "1.14"
```

Comment:
Since v1.10 is still supported, I'd rather keep it in `nginx-rhel7.json` and `nginx-centos7.json` image stream.

Reply:
Good catch. Typo.  I will fix that